lib/ex_doc/formatter/html/templates/detail_template.eex

```diff
@@ -1,4 +1,4 @@
-<div class="detail" id="<%=enc_h link_id(node) %>">
+<div class="detail" id="<%= link_id(node) %>">
```

Comment:
Can we call `h link_id(node)` instead? We don't need to encode, but we still need to escape it, right? :) Thank you! :heart: